src/user-ratings/UserRatings.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import "./UserRatings.css"
+import SingleMovie from '../single-movie/SingleMovie.js'
+import { getMovieRatings } from '../apiCalls.js'
+
+
+export default class UserRatings extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedRating: null,
+      error: ''
+    }
+    this.changeValue = this.changeValue.bind(this);
+  }
+
+  fetchMovieRatingsData(userId, movieId, ratings) {
+    console.log('hello')
+    getMovieRatings(userId, movieId, ratings)
+    .then(() => this.props.fetchUserRatings(userId))
```

Comment:
the .then(() => get fetch) is a great way to make sure the post request is resolved before getting that data again